chia/wallet/puzzles/custody/member_puzzles/secp256r1_member.clsp

```diff
@@ -0,0 +1,11 @@
+; this puzzle follows the Managed Inner Puzzle Spec MIPS01 as a Member Puzzle
+; this code offers a secure approval of a delegated puzzle passed in as a Truth to be run elsewhere
+
+(mod (SECP_PK Delegated_Puzzle_Hash my_id signature)  ; delegated puzzle is passed in from the above M of N layer
```

Comment:
In order to perfectly mimic AGG_SIG_ME, we also need the network genesis challenge.

Reply:
Offline discussion provided me with the insight that currying in the genesis challenge doesn't actually do anything for us here.  Maybe we should look at removing this from our puzzles that we have ported from.